src/oop/classes/Session.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+require_once 'Cookie.php';
+
+class Session extends Cookie
+{
+	
+    public $placeholder; //session array
+
+    public function __construct($session)
+    {
+        $this->placeholder = $session;
+    }
+
+    public function set($key, $value = 'default')
+    {
+        $this->placeholder = array_merge($this->placeholder, [$key => $value]);
+        return "Data with the $value value was created under the $key key.";
```

Comment:
it is better to trow exception

Reply:
I made this string-return only for the method's working visualization on the web page.
Should I still replace it by throwing an exception or just delete the return statement at all?

Reply:
You should always separate logic and representation and one class should not have both. That is why you should not use strings as return